src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task{
```

Comment:
Whitespace should proceed { , according to our coding standard. Just something to take note.